apps/src/templates/sectionProgress/ProgressViewHeader.jsx

```diff
@@ -45,6 +45,17 @@ class ProgressViewHeader extends Component {
     return scriptData ? `${scriptData.path}?section_id=${section.id}` : null;
   }
 
+  getLinkToStandardsOverview() {
```

Comment:
would it make more sense for this to live inside CreateStandardsReportStep2, since that's where it's used? It seems like that would be cleaner since we wouldn't have to pass it as a prop through the intermediate layers

Reply:
Done!